### ISSUE TITLE
[a11y] 9.1.3.1h Labeling of form elements can be determined programmatically

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10724]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-3-1h-beschriftung-von-formularelementen-programmatisch-ermittelbar

Programmatic linking of labels with form elements takes place with
"label" elements via the "for" attribute or the inclusion of the labeled form element in the
form element in the "label" element. If labels are not marked with the "label" element
element, a label for the associated form element should be created in another way
other way (e.g. via the "aria-labelledby" attribute).

### Personal: Partially fulfilled.

- [ ] When focusing the slider "Tile size" in the menu "Customization options for the
Show file list customization options" menu, the accessible name is not displayed because the
programmatic linking using the "for"/"id" attributes does not work. The
corresponding "id" attribute is missing in the "input" element.
![image](https://github.com/owncloud/web/assets/132444655/c8e3db5b-6c75-4b39-b50f-c19b9a384e37)

### Spaces: Partially fulfilled.

- [ ]  The "Filter members" field in the "Members" section is not accessible as it is not programmatically
is not programmatically linked to a label. Although a "label"
element is present, it does not contain any text. A placeholder is not a suitable
replacement for a label, as it is not output in many cases. 
![image](https://github.com/owncloud/web/assets/132444655/10d81574-5284-42b8-b4c3-ae58e7a5a8a7)

- [ ] See commentary on Personal

Info: Shares, Deleted Files and Account: Fulfilled
